### DISCUSSION TITLE
no new builds?

### DISCUSSION THREAD

**Discussion by hb0nd** [grote/Transportr#821] [General]
There haven't been any new builds for a long while, is the project winding down?

**Comment by ialokim** [comment]
We are only two developers working in our free time on this app, so indeed progress has been quite slow lately. Right now we've also faced problems with our distribution channels ([F-Droid](https://gitlab.com/fdroid/fdroidserver/-/issues/1004#note_1030833616) and [Google Play](https://github.com/grote/Transportr/issues/811)), so we first need to find some time to work on those to even be able to publish new builds at all.

**Comment by Altonss** [comment]
An effort to try a release a new build and developement pick up more speed is being discussed here: https://github.com/grote/Transportr/issues/852 :)
A small new release is planned hopefully soon: here is the discussion for it https://github.com/grote/Transportr/discussions/854
